packages/bpk-component-navigation-bar/src/BpkNavigationBar.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cssModules } from 'bpk-react-utils';
+import BpkText from 'bpk-component-text';
+
+import STYLES from './bpk-navigation-bar.scss';
+
+const getClassNames = cssModules(STYLES);
+
+type Props = {
+  title: string,
+  className: ?string,
+  leadingButton: ?React$Element<any>,
+  trailingButton: ?React$Element<any>,
+};
+
+const cloneWithStyle = (elem: React$Element<any>, newStyle: string) => {
+  const className = getClassNames(elem.props.className, newStyle);
+  return React.cloneElement(elem, { ...elem.props, className });
+};
+
+const BpkNavigationBar = (props: Props) => {
+  const { title, className, leadingButton, trailingButton } = props;
+
+  return (
+    <nav
+      aria-labelledby="bpk-navigation-bar-title"
+      className={getClassNames('bpk-navigation-bar', className)}
+    >
+      {leadingButton &&
+        cloneWithStyle(leadingButton, 'bpk-navigation-bar__leading-item')}
+      <BpkText
+        id="bpk-navigation-bar-title"
```

Comment:
this ID should be customised by userland i.e `id={``${props.id}-navigation-bar-title``}`